templates/docs/examples/patterns/tables/table-mobile-card.html

```diff
@@ -4,7 +4,7 @@
 {% block standalone_css %}patterns_table-mobile-card{% endblock %}
 
 {% block content %}
-<table class="p-table--mobile-card" aria-label="Example of table transforming into mobile cards on small screens">
+<table class="p-table--mobile-card is-split" aria-label="Example of table transforming into mobile cards on small screens">
```

Comment:
Does `is-split` do anything? I don't see any styling specific for it.

Reply:
no, removed